composer.json

```diff
@@ -26,10 +26,11 @@
 		"roave/security-advisories": "dev-latest",
 		"phpunit/phpunit": "^9.6",
 		"psalm/phar": "^5.15",
-		"friendsofphp/php-cs-fixer": "3.38"
+		"friendsofphp/php-cs-fixer": "3.38",
+		"php-parallel-lint/php-parallel-lint": "^1.3"
 	},
 	"scripts": {
-		"lint": "find . -name \\*.php -not -path './vendor/*' -print0 | xargs -0 -n1 php -l",
+		"lint": "parallel-lint --exclude vendor .",
```

Comment:
Is introducing this new dependency really necessary? :thinking:

Reply:
not really, but it's fast and it's a dev one so I figured why not? I can remove the commit it you want, nothing hard and fast about it.

Reply:
I'm rather fond of the simplicity of the previous solution